spec/sqewer/local_connection_spec.rb

```diff
@@ -87,4 +87,14 @@
     Process.wait(consumer_pid)
     expect($?.exitstatus).to eq(0)
   end
+
+  it 'allows using an in memory database' do
+    conn = described_class.new('sqlite3://memory')
```

Comment:
The example in the README uses one `/` like this: `sqlite3:/memory`

Reply:
tks, the correct is using `//` otherwise `URI.parse` does not work. I fixed in the README